source/edge-server/permissions.txt

```diff
@@ -0,0 +1,266 @@
+permissions to configure an Edge Server instance to read data, but not write
+it.
+
+Because every Edge Server instance is a user, you can use the Edge Server's
+``user_id`` to configure permissions for a given instance. You can create a 
+role that restricts the data that syncs with a specific Edge Server instance
+using a :ref:`rule expression <expressions>` where the 
+:ref:`%%user_id <rule-expression-user-expansion>` is the Edge Server's 
+``user_id``. You can get the Edge Server instance ``user_id`` from the
+:ref:`edge-server-details`, or from the Admin API. [PLACEHOLDER - Add link 
+to the relevant Admin API endpoint when the docs PR is merged.]
+
+Read/Write Own Documents
+++++++++++++++++++++++++
+
+You can restrict an Edge Server instance to read and write only its own 
+documents using the ``user_id``. For example, if an Edge Server instance 
+represents a health care clinic, you might only want to sync data relevant to 
+that clinic's patients:
+
+.. code-block:: json
+
+   {
+     "name": "facilityItemsOnly",
+     "apply_when": {},
+     "document_filters": {
+       "write": {
+         "facility_id": "%%user.id"
+       },
+       "read": {
+         "facility_id": "%%user.id"
+       },
+     },
+     "read": true,
+     "write": true,
+     "insert": true,
+     "delete": true,
+     "search": true
+   }
+
+In this example, every document has a ``facility_id`` property whose value is
+the Edge Server's ``user_id``. This role means that the Edge Server can only
+read and write documents with a ``facility_id`` matching its own ``user_id``.
+
+Read All, Write Own
++++++++++++++++++++
+
+You can configure an Edge Server instance to be able to read all data, but
+write only its own data. For example, in a retail setting, an Edge Server 
+instance might represent a store inventory system. You might want the store to
+have the ability to search across all inventory for an item, but only sell 
+items in its own inventory.
+
+.. code-block:: json
+
+   {
+     "name": "readAllWriteOnlyStoreItems",
+     "apply_when": {},
+     "document_filters": {
+       "write": {
+         "store_id": "%%user.id"
+       },
+       "read": true
+     },
+     "read": true,
+     "write": true,
+     "insert": true,
+     "delete": true,
+     "search": true
+   }
+
+Client Permissions
+~~~~~~~~~~~~~~~~~~
+
+Clients that connect to an Edge Server instance have their own permissions. 
+You can think of an Edge Server instance's permissions as a filter between the
+data in Atlas and the data that a client can read and write.
+
+In a healthcare setting similar to the Edge server instance example above, the
+Edge Server instance may only read and write data that is relevant to its own 
+facility. A client may then further restrict the permissions. For example, a 
+patient in the facility may only view their own data.
+
+You represent these as independent roles within the ``roles`` array.
```

Comment:
```suggestion
You can represent these as independent roles within the ``roles`` array.
```

Reply:
I avoided "can" here because that brings along an implied optionality, but I'm trying to communicate that this isn't optional - you define Edge Server permissions and client permissions as independent roles within the array.